### ISSUE TITLE
Issue with release 1.2

### ISSUE THREAD

**Issue by cccs-jc** [joe-re/sql-language-server#118]
There seems to be an issue with release 1.2.

I've installed it like so

```
npm install -g sql-language-server
```

I'm getting this error when JupyterLab communicates with the sql-language-server
```
 [ERROR] default - uncaughtException TypeError: server_1.TextDocuments is not a constructor
    at createServerWithConnection (/opt/conda/lib/node_modules/sql-language-server/dist/src/createServer.js:48:21)
    at createServer (/opt/conda/lib/node_modules/sql-language-server/dist/src/createServer.js:311:12)
    at Object.handler (/opt/conda/lib/node_modules/sql-language-server/dist/bin/cli.js:25:37)
    at Object.runCommand (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/lib/command.js:238:44)
    at Object.parseArgs [as _parseArgs] (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/yargs.js:1063:30)
    at Object.get [as argv] (/opt/conda/lib/node_modules/sql-language-server/node_modules/yargs/yargs.js:1004:21)
    at Object.<anonymous> (/opt/conda/lib/node_modules/sql-language-server/dist/bin/cli.js:30:5)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
```

I checked the version of vscode-languageserver using

```
npm list -g --depth 2 
```
it is at version vscode-languageserver@8.0.0-next.8

this version is different than the one specified in the source code which is vscode-languageserver@8.0.0-next.3

And when I look at the vscode-languageserver@8.0.0-next.8 version the file /opt/conda/lib/node_modules/sql-language-server/node_modules/vscode-languageserver/lib/common/server.js 

does not contain a `TextDocuments` class

there seems to be a version issue with the npm package. I'm not sure how to resolve this issue.

**Comment by joe-re** [comment]
@cccs-jc Hi, thanks for your report. Let me try to fix the problem.

**Comment by cccs-nik** [comment]
Hello @joe-re,

I took a look on behalf of jc, the fix seems to be pretty simple.
https://github.com/joe-re/sql-language-server/blob/release/packages/server/package.json#L53
That package's version should be '^8.0.0-next.7' and this import:
https://github.com/joe-re/sql-language-server/blob/release/packages/server/src/createServer.ts#L7
should be 'vscode-languageserver/lib/common/textDocuments'.

I think that's it. I tried pushing my branch but I don't have the correct permissions to do so.

**Comment by joe-re** [comment]
@cccs-jc @cccs-nik Hi, thanks for your guy's support. I fixed the problem and released v1.2.1. Please check it out and confirm it works. Thanks. 🙏

**Comment by cccs-nik** [comment]
Looks good on our end. Thanks for the quick fix!
